build/scripts/tools.ps1

```diff
@@ -413,7 +413,7 @@ function MSBuild() {
   $cmdArgs = "$($buildTool.Command) /m /nologo /clp:Summary /v:$verbosity /nr:$nodeReuse"
 
   if ($warnAsError) { 
-    $cmdArgs += " /warnaserror /p:TreatWarningsAsErrors=true" 
+    $cmdArgs += " /p:TreatWarningsAsErrors=true" 
```

Comment:
@tmat I had to make this change or the build would ignore my request for `<WarningsNotAsErrors>`.